.travis.yml

```diff
@@ -31,8 +31,9 @@ before_install:
   - sudo apt-get -q update
 
 install:
-  - sudo apt-get install -y python
+  - sudo apt-get install -y python python3 python3-pip
```

Comment:
While it is not an issue on our test setup, `python` might be an alias of python3 on other distros. I would be better to specify `python2` explicitly.